src/ledger/ledger-txn-application-call.md

```diff
@@ -35,9 +37,10 @@ An _application call_ transaction additionally has the following fields:
 | Foreign Applications | `apfa` | `[]uint64`  |            No             |
 | Foreign Assets       | `apas` | `[]uint64`  |            No             |
 | Box References       | `apbx` | `[]struct`  |            No             |
+| Access List          |  `al`  | `[]struct`  |            No             |
```

Comment:
`[]struct` is not very informative.

Reply:
Struct fields are defined and described in a table after, what would you change?